### ISSUE TITLE
Nombre Repetido con distinto ID

### ISSUE THREAD

**Issue by santiagoBassi** [PI-ITBA/TPE#35]
Hola,  buenas noches. Queria avisar que en el archivo "stationsMON.csv" hay dos estaciones con un mismo nombre y dos id diferentes. Lineas 916 y 45. La estacion se llama "Métro Pie-IX (Pierre-de-Coubertin / De La Salle)"

**Comment by fmeola** [comment]
Hola Santiago.
Sí, te referís a:

```
pk;name;latitude;longitude
1010;Métro Pie-IX (Pierre-de-Coubertin / de la Salle);45.5552411;-73.5508221
907;Métro Pie-IX (Pierre-de-Coubertin / De La Salle);45.5552411;-73.5508221

```
Los nombres difieren porque después de la barra la primera tiene "de la" en minúscula y la segunda tiene "De La" con mayúscula.

Recuerden lo aclarado en https://github.com/PI-ITBA/TPE/issues/13

**Comment by santiagoBassi** [comment]
Ah bueno, muchas gracias. 
Aprovecho a hacerte otra consulta, en la query dos se pide devolver ordenado alfabeticamnete, y vos dijiste que en los nombres puede haber simbolos, cual seria el sentido de orden ahi?

**Comment by santiagoBassi** [comment]
Nosotros nos guiamos por lo que dijo marcelo en #10 pero a lo mejor lo interpretamos mal. Al usar strcasecmp se nos rompe con esa estación

**Comment by fmeola** [comment]
No, está bien. En el caso de usar `strcasecmp` entonces las estaciones en conflicto de MON son 3:

```
métro pie-ix (pierre-de-coubertin / de la salle)
de lanaudière / du mont-royal
de la commune / king
```

En el set de NYC no hay estaciones en conflicto.

Pueden entonces ustedes editar el CSV de estaciones de MON y agregarle un `(2)` al nombre de la estación de una de las dos para los tres pares. Por ejemplo:

```
pk;name;latitude;longitude
1010;Métro Pie-IX (Pierre-de-Coubertin / de la Salle);45.5552411;-73.5508221
907;Métro Pie-IX (Pierre-de-Coubertin / De La Salle) (2);45.5552411;-73.5508221

```

Nosotros en la corrección vamos a utilizar un CSV que no tenga dos estaciones con nombres iguales (ahora también sin importar mayúsculas o minúsculas)

Respecto a lo del orden alfabético para símbolos, no hay que contemplar nada especial. Es el orden de `strcmp` o `strcasecmp`.
